lib/terraforming/version.rb

```diff
@@ -1,3 +1,3 @@
 module Terraforming
-  VERSION = "0.9.1"
+  VERSION = "0.10.0"
```

Comment:
Please do not bump the version in this pull request.

If you'd like to install this patch locally, just run `rake install`.